<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>588</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>ToDoName</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMembers">
    <property name="title">
     <string>Members</string>
    </property>
    <addaction name="actionAdd_New"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuLoan">
    <property name="title">
     <string>Loan</string>
    </property>
    <addaction name="actionNew_Loan"/>
    <addaction name="actionView_Loans"/>
   </widget>
   <widget class="QMenu" name="menuTransaction">
    <property name="title">
     <string>Transaction</string>
    </property>
    <addaction name="actionDeposit"/>
    <addaction name="actionWithdraw"/>
    <addaction name="actionTransfer"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <widget class="QMenu" name="menuAccounts">
     <property name="title">
      <string>Accounts</string>
     </property>
     <addaction name="actionRecurring_Deposit"/>
     <addaction name="actionFixed_Deposit"/>
     <addaction name="actionSavings"/>
    </widget>
    <widget class="QMenu" name="menuLoan_2">
     <property name="title">
      <string>Loan</string>
     </property>
     <addaction name="actionAdvance"/>
     <addaction name="actionOutstanding"/>
    </widget>
    <addaction name="menuAccounts"/>
    <addaction name="menuLoan_2"/>
    <addaction name="actionBalance_Sheet"/>
    <addaction name="actionProfit_and_Loss"/>
    <addaction name="actionReceipt_and_Disbursment"/>
    <addaction name="actionShareholders"/>
    <addaction name="actionTerm_Deposit_Account"/>
    <addaction name="actionMisc"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuBackup">
    <property name="title">
     <string>Backup</string>
    </property>
    <addaction name="actionRestore"/>
    <addaction name="actionSync"/>
   </widget>
   <widget class="QMenu" name="menuExpense">
    <property name="title">
     <string>Expense</string>
    </property>
    <addaction name="actionNew_Expense"/>
    <addaction name="actionView_Expense"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionSupport"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <addaction name="menuMembers"/>
   <addaction name="menuLoan"/>
   <addaction name="menuTransaction"/>
   <addaction name="menuReport"/>
   <addaction name="menuExpense"/>
   <addaction name="menuBackup"/>
   <addaction name="menuAbout"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_New">
   <property name="text">
    <string>Add New</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find Member</string>
   </property>
  </action>
  <action name="actionNew_Loan">
   <property name="text">
    <string>New Loan</string>
   </property>
  </action>
  <action name="actionView_Loans">
   <property name="text">
    <string>View Loan</string>
   </property>
  </action>
  <action name="actionVoucher">
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
  <action name="actionPay_Slip">
   <property name="text">
    <string>Withdraw</string>
   </property>
  </action>
  <action name="actionBalance_Sheet">
   <property name="text">
    <string>Balance Sheet</string>
   </property>
  </action>
  <action name="actionProfit_and_Loss">
   <property name="text">
    <string>Profit and Loss</string>
   </property>
  </action>
  <action name="actionReceipt_and_Disbursment">
   <property name="text">
    <string>Receipt and Disbursment</string>
   </property>
  </action>
  <action name="actionShareholders">
   <property name="text">
    <string>Shareholders</string>
   </property>
  </action>
  <action name="actionTerm_Deposit_Account">
   <property name="text">
    <string>Term Deposit Accounts</string>
   </property>
  </action>
  <action name="actionStationary">
   <property name="text">
    <string>Stationary</string>
   </property>
  </action>
  <action name="actionFurniture">
   <property name="text">
    <string>Furniture</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
  <action name="actionRecurring_Deposit">
   <property name="text">
    <string>Recurring Deposit</string>
   </property>
  </action>
  <action name="actionFixed_Deposit">
   <property name="text">
    <string>Fixed Deposit</string>
   </property>
  </action>
  <action name="actionSavings">
   <property name="text">
    <string>Savings </string>
   </property>
  </action>
  <action name="actionStationary_2">
   <property name="text">
    <string>Stationary</string>
   </property>
  </action>
  <action name="actionAdvance">
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="actionOutstanding">
   <property name="text">
    <string>Outstanding</string>
   </property>
  </action>
  <action name="actionBank">
   <property name="text">
    <string>Bank</string>
   </property>
  </action>
  <action name="actionNew_Expense">
   <property name="text">
    <string>New Expense</string>
   </property>
  </action>
  <action name="actionView_Expense">
   <property name="text">
    <string>View Expense</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string>Support</string>
   </property>
  </action>
  <action name="actionMisc">
   <property name="text">
    <string>Misc</string>
   </property>
  </action>
  <action name="actionDeposit">
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
  <action name="actionWithdraw">
   <property name="text">
    <string>Withdraw</string>
   </property>
  </action>
  <action name="actionTransfer">
   <property name="text">
    <string>Transfer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
